css/css-anchor-position/position-visibility-no-overflow-scroll-002.html

```diff
@@ -11,7 +11,7 @@
     overflow: hidden;
   }
   #scroll-container {
-    overflow: hidden scroll;
+    overflow: hidden;
```

Comment:
```suggestion
    overflow: hidden scroll;
```

Reply:
So this makes us flaky on these tests - I'll keep as hidden.